validator/rpc/standard_api_test.go

```diff
@@ -724,23 +728,60 @@ func TestServer_ListFeeRecipientByPubkey(t *testing.T) {
 			wantErr: false,
 		},
 		{
-			name: "empty settings",
+			name: "Happy Path Test Cached",
```

Comment:
```suggestion
			name: "happy path test cached",
```
other test cases use small caps